packages/integration-karma/test/rendering/slot-dont-recycle-vm/index.spec.js

```diff
@@ -0,0 +1,30 @@
+import { createElement } from 'lwc';
+import Container from 'x/container';
+
+describe('Component', () => {
```

Comment:
The name of this file (which is also used as part of the test name) doesn't seem accurate. It should probably be something along the lines of `rendering/elements-are-not-recycled`.

Reply:
It doesn't look like we need this `describe`.